backend/substrapp/tasks/utils.py

```diff
@@ -220,6 +220,87 @@ def list_files(startpath):
         logger.info(f'{startpath} does not exist.')
 
 
+def build_docker_image(dockerfile_path, image_name):
+    dockerfile_fullpath = os.path.join(dockerfile_path, 'Dockerfile')
+    current_namespace = os.getenv('NAMESPACE')
```

Comment:
You could get the namespac from the kubernetes python client :)

Reply:
The only solution I found to do this does not work with the `config.load_incluster_config()` as we only get a token to interact with the API and not a full kubeconfig with context informations like the namespace.
As the env workaround was already used [here](https://github.com/SubstraFoundation/substra-backend/blob/master/backend/substrapp/tasks/tasks.py#L757) I figured that I should use it too.

Reply:
Yes, you're right, maybe rename this variable to make it explicit that this namespace will be use for that ! :)